### PR TITLE
LVGL: Updated BSPs and examples for new ESP LVGL Port (LVGL9)

### PR DESCRIPTION
**Pull request by espzav**
# ESP-BSP Pull Request checklist

- [x] Version of modified component bumped
- [x] CI passing

# Change description

Wait for merge #268

# Examples tested in LVGL8
- [ ] display
- [ ] display_audio_photo
- [ ] display_camera
- [ ] display_lvgl_demos
- [ ] display_rotation
- [ ] display_sensors
- [ ] display_usb_hid

# Examples tested in LVGL9
- [x] display
- [x] display_audio_photo
- [x] display_camera
- [x] display_lvgl_demos
- [x] display_rotation
- [x] display_sensors
- [x] display_usb_hid

### CODE REVIEW

**Review by tore-espressif — APPROVED** [review]
@espzav I left one comment about SquareLineStudio

The rest should be similar to #268 , LGTM!

File: SquareLine/boards/esp32_c3_lcdkit/sdkconfig.defaults [hunk 1]
```diff
@@ -1,5 +1,2 @@
 CONFIG_IDF_TARGET="esp32c3"
-CONFIG_LV_COLOR_16_SWAP=y
-CONFIG_LV_MEM_CUSTOM=y
-CONFIG_LV_MEMCPY_MEMSET_STD=y
```
[the remark this answers is not in the document]
Comment (espzav) [reply SquareLine/boards/esp32_c3_lcdkit/sdkconfig.defaults]: Thanks for the warn me. I didn't check SquareLine release yet.

File: bsp/esp32_s3_eye/include/bsp/esp32_s3_eye.h [hunk 2]
```diff
@@ -383,7 +383,7 @@ esp_err_t bsp_display_backlight_off(void);
  * @param[in] disp Pointer to LVGL display
  * @param[in] rotation Angle of the display rotation
  */
-void bsp_display_rotate(lv_disp_t *disp, lv_disp_rot_t rotation);
+void bsp_display_rotate(lv_display_t *disp, lv_disp_rotation_t rotation);
```
Comment (tore-espressif) [inline bsp/esp32_s3_eye/include/bsp/esp32_s3_eye.h:386]: @espzav Without releasing this change, the BSP won't compile, right?

Comment (tore-espressif) [reply bsp/esp32_s3_eye/include/bsp/esp32_s3_eye.h:386]: @espzav I think we can 'yank' esp_lvgl_port v1.4.0 so this problem does not propagate to user's projects and then we must fix

Comment (espzav) [reply bsp/esp32_s3_eye/include/bsp/esp32_s3_eye.h:386]: @tore-espressif Oh, I forgot increase versions of BSPs. I am stupid. I can make new PR with increase all BSPs versions - are you agree?

Comment (tore-espressif) [reply bsp/esp32_s3_eye/include/bsp/esp32_s3_eye.h:386]: There were more issues with `esp_lvgl_port` 1.5.0 so it is yanked know. Let's resolve all build issues and start again :)

**Pull request merged**